src/Controller/DefaultRouteController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\BannerServer\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class DefaultRouteController {
+	public function index( Request $request ): Response {
```

Comment:
didn't see that the first time, sorry, but that Request parameter can also go,
other than that it looks good to me